### DISCUSSION TITLE
Making an Operating System in C.

### DISCUSSION THREAD

**Discussion by OSEight** [community/community#73164] [Programming Help]
### Body

Hello! I am making an operating system and would like some help. It is called OS Eight. Ya'll can find it at OSEight/OSEightStructure.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by davea42** [comment]
Sorry, I'm unable to find any relevant content in your projects.  Which is fine, but makes it impossible to be any help at all. Sorry.  Maybe I just don't get it.

**Reply by OSEight** [reply]
Okay, I'll make the repository public.
